rpmlint/checks/SysVInitOnSystemdCheck.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class SysVInitOnSystemdCheck(AbstractCheck):
+    def check(self, pkg):
+        if pkg.isSource():
+            return
+
+        for req in pkg.requires() + pkg.prereq():
+            if req[0] == 'insserv':
+                self.output.add_info('E', pkg, 'obsolete-insserv-requirement')
+
+        for filename in pkg.files():
+            if not filename.startswith('/etc/init.d'):
```

Comment:
You will need to add `/etc/rc.d` to this. RH-ish systems installed to `/etc/rc.d/init.d`, per `%_initddir`:

```
[ngompa@localhost ~]$ rpm -E "%_initddir"
/etc/rc.d/init.d
```

Reply:
Thanks, patch has been updated.